osmosis-1/osmosis-1.assetlist.json

```diff
@@ -8065,6 +8065,49 @@
         "osmosis-info",
         "OSMO:969"
       ]
-    }
+    },
+    {
+      "description": "The native token of OmniFlix Hub chain",
+      "denom_units": [
+        {
+          "denom": "ibc/CEE970BB3D26F4B907097B6B660489F13F3B0DA765B83CC7D9A0BC0CE220FA6F",
+          "exponent": 0,
+          "aliases": [
+            "uflix"
+          ]
+        },
+        {
+          "denom": "flix",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/CEE970BB3D26F4B907097B6B660489F13F3B0DA765B83CC7D9A0BC0CE220FA6F",
+      "name": "OmniFlix Network",
+      "display": "flix",
+      "symbol": "FLIX",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "omniflixhub",
+            "base_denom": "uflix",
+            "channel_id": "channel-1"
+          },
+          "chain": {
+            "channel-id": "channel-199",
```

Comment:
I think it's failing because this is meant to be an underscore. But you can actually delete the changes to the osmosis-1.assetlist.json file since that's an automatically generated file; we just  need the addition to the zone file.

Reply:
Oh okay
 got it